<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="1" failures="0" skipped="0" tests="1" time="0.019" timestamp="2023-08-08T19:18:40.699080" hostname="propcik"><testcase classname="tests.xmls.test_error_in" name="test_error" time="0.000"><error message="failed on setup with &quot;RuntimeError&quot;">@pytest.fixture
    def my_fixture():
&gt;       raise RuntimeError()
#x1B[1m#x1B[31mE       RuntimeError#x1B[0m

#x1B[1m#x1B[31mtest_error_in.py#x1B[0m:6: RuntimeError</error></testcase></testsuite></testsuites>